README.md

```diff
@@ -7,11 +7,12 @@
 - Select executable installer from the options that appear.
 
 We used Python 3.6.4 to make our stuff.
+Also we used an IDE but Python comes with IDLE. Just saying IDE is better.
```

Comment:
My suggestions...

Reply:
Maybe change to  `We built this on VSCode which you should check out, it's a great IDE!`